calico_node/startup/startup.go

```diff
@@ -282,12 +289,15 @@ func parseIPEnvironment(envName, envValue string, version int) *net.IP {
 
 // validateIP checks that the IP address is actually on one of the host
 // interfaces and warns if not.
-func validateIP(ip *net.IP) {
+func validateIP(ipn *net.IPNet) {
```

Comment:
Could argue both ways.  The field in the API is just called IPAddress rather than CIDR *and* we are actually only validating the IP address (we aren't validating the network part).

Given the latter point, I think it's ok as is.